packages/react-core/src/components/EmptyState/EmptyState.tsx

```diff
@@ -10,37 +11,78 @@ export enum EmptyStateVariant {
   full = 'full'
 }
 
+export type EmptyStateStatus = 'danger' | 'warning' | 'success' | 'info' | 'custom';
+export type EmptyStateVariantType = 'xs' | 'sm' | 'lg' | 'xl' | 'full';
+
 export interface EmptyStateProps extends React.HTMLProps<HTMLDivElement> {
   /** Additional classes added to the empty state */
   className?: string;
   /** Content rendered inside the empty state */
-  children: React.ReactNode;
+  children?: React.ReactNode;
```

Comment:
`children` changed to optional because `titleText` has been lifted and changed to required.